src/display/editor/freetext.js

```diff
@@ -322,12 +340,18 @@ class FreeTextEditor extends AnnotationEditor {
 
     super.render();
     this.editorDiv = document.createElement("div");
-    this.editorDiv.tabIndex = 0;
     this.editorDiv.className = "internal";
 
-    FreeTextEditor._l10nPromise.then(msg =>
-      this.editorDiv.setAttribute("default-content", msg)
-    );
+    this.div.setAttribute("role", "textbox");
+    this.div.setAttribute("aria-multiline", true);
+
+    FreeTextEditor._l10nPromise
+      .get("editor_freetext_aria_label")
+      .then(msg => this.div.setAttribute("aria-label", msg));
```

Comment:
Given the asynchronicity of the code, would it be safer to do the following?
```suggestion
      .then(msg => this.div?.setAttribute("aria-label", msg));
```